src/include/sof/drivers/dw-dma.h

```diff
@@ -155,10 +150,10 @@ struct dw_lli {
 	uint32_t sstat;
 	uint32_t dstat;
 
-	/* align to required bytes to make sure every item
-	 * is aligned in case of more than two items
+	/* align to 32 bytes to not cross cache line
+	 * in case of more than two items
 	 */
-	uint8_t reserved[DW_DMA_LLI_ALIGN - sizeof(uint32_t) * 7];
+	uint32_t reserved;
```

Comment:
what does this have to do with this PR? this seems completely unrelated to the use of HP SRAM banks?